cpp/modmesh/universe/World.hpp

```diff
@@ -99,10 +119,17 @@ class World
         }
     }
 
+    std::deque<Edge3d<T>> m_edges;
```

Comment:
I am using a `deque`.  But later I will create a dedicated container for `Edge` and `Bezier`.